idea_town/frontend/static-src/styles/main.scss

```diff
@@ -1,11 +1,8 @@
-@import '../vendor/normalize.css/normalize';
-
-// pull in tabzilla just so the build tools see it
+@import 'normalize';
```

Comment:
Not sure if we want to move the `normalize` and `tabzilla.scss` in the main.scss or move it into a `_vendor.scss` file, so we can just do an `@import 'vendor';` here.

On a side note, I have a forked branch of Tabzilla which makes it easier to use w/ Sass so we wouldn't need to move into a /vendor/ directory and do a bunch of steps. We'd just need to hack the package.json to use [**pdehaan/tabzilla#sass-alternate**](https://github.com/pdehaan/tabzilla/tree/sass-alternate).
I'll ping the tabzilla people again and see if there is still interest in merging my PR so this would be in the tabzilla official repo.